core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/AzureBatchSource.java

```diff
@@ -61,7 +61,6 @@ public AzureBatchSource(AzureBatchConfig config) {
     @Override
     protected Map<String, String> getFileSystemProperties() {
       Map<String, String> properties = new HashMap<>(super.getFileSystemProperties());
-      properties.put("fs.defaultFS", String.format("wasb://%s@%s/", container, account));
```

Comment:
Here is "[Use HDFS-compatible storage with Hadoop in HDInsight](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-use-blob-storage)" .  The  syntax is wasb[s]://<containername>@<accountname>.blob.core.windows.net/<path>.  The recommendation is to use wasbs wherever possible.